core/src/main/java/org/apache/iceberg/rest/responses/LoadViewResponse.java

```diff
@@ -34,11 +31,6 @@ public interface LoadViewResponse extends RESTResponse {
 
   Map<String, String> config();
 
-  @Value.Default
```

Comment:
As the `credentials()` method didn't exist in previous releases (it has been introduced recently), and as it can be problematic for "secure views", I think it makes sense to remove it before 1.7 release.